common/lib/xmodule/xmodule/x_module.py

```diff
@@ -278,6 +278,10 @@ class XModuleMixin(XModuleFields, XBlock):
     # (like a practice problem).
     has_score = False
 
+    # Whether this module can be displayed in read-only mode.  It is safe to set this to True if
+    # all user state is handled through the FieldData API.
+    show_in_read_only_mode = False
```

Comment:
I guess my thought was that an XBlock that stores state externally might still have a view that wouldn't affect that state, and thus should be eligible for masquerading.

But for now, this is fine, I think.